### DISCUSSION TITLE
[FEATURE] Spawn on join

### DISCUSSION THREAD

**Discussion by SkyDude0** [Zrips/CMI#6900] [Enhancement]
Add option where you can choose on witch world you spawn at spawn point when join.

Ex. If i add 'dungeon' world to the config, only when i leave the server on dungeon world when players re-join, spawn at normal spawn point

**Comment by Zrips** [comment]
This is a thing already, use something like **/setspawn -w:LT_Craft_nether** which will put player into this spawn point if he logged off in nether world, you can specify more than one world by separating those with comma **/setspawn -w:world_nether,world**

**Comment by Zrips** [comment]
Provided list of worlds will define from where player gets teleported to this spawn point

**Reply by SkyDude0** [reply]
yea, i delete my message, i didnt read the last thing. Ill test it soon

**Reply by SkyDude0** [reply]
it dosent work, im trying now

i try with: /setspawn -w:test , but with relog , nothing happend

**Reply by SkyDude0** [reply]
the problem is that thing that i want is impossible to do now. What I need is a double system, after in some worlds after you leave and re-enter the server you find yourself in the same position, in others I need the possibility to add a spawn point like the one that already exists. Such a double system is currently not possible, I have tried everything but still I can not do what I need. Hope you consider adding this dual system someday
